app/uk/gov/hmrc/securemessage/controllers/SecureMessageController.scala

```diff
@@ -38,7 +42,12 @@ class SecureMessageController @Inject()(
   def createConversation(client: String, conversationId: String): Action[JsValue] = Action.async(parse.json) {
     implicit request =>
       withJsonBody[ConversationRequest] { conversationRequest =>
-        secureMessageService.createConversation(conversationRequest, client, conversationId)
+        secureMessageService.createConversation(conversationRequest.asConversation(client, conversationId)).map {
```

Comment:
love this clear pattern: 
call service, map result to specific Http Responses. 
Good job :)